src/components/TodoItem.tsx

```diff
@@ -16,25 +16,36 @@ const TodoItem = (props: TodoItemProps) => {
   const { text, creationDate, expirationDate, id, done } = props.item;
   const dispatch = useDispatch();
 
-  const togglehandler = () => {
-    dispatch(todosActions.toggleTodo(id));
-  };
+  const toggleHandler = () => dispatch(todosActions.toggleTodo(id));
+  const deleteHandler = () => dispatch(todosActions.removeTodo(id));
 
   const crossed = `${done ? classes.crossed : ''}`;
 
+  const { item, input, texT, other, icons, crossmark, dates, date, header } =
+    classes;
+
   return (
-    <li className={classes.item}>
-      <input type="checkbox" checked={done} onChange={togglehandler} />
-      <div className={`${classes.text} ${crossed}`}>{text}</div>
-      <div className={classes.other}>
-        <div className={classes.icons}></div>
-        <div className={`${classes.dates} ${crossed}`}>
-          <div className={classes.date}>
-            <span className={classes.header}>Created: </span>
+    <li className={item}>
+      <input
+        type="checkbox"
+        checked={done}
+        onChange={toggleHandler}
+        className={input}
+      />
+      <div className={`${texT} ${crossed}`}>{text}</div>
+      <div className={other}>
+        <div className={icons}>
+          <button className={crossmark} onClick={deleteHandler}>

```

Comment:
Don't you want to use svg for icon ?

Reply:
Is it a letter? Because seems like some symbol

Reply:
symbol)